src/main/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorConfigService.java

```diff
@@ -42,18 +48,37 @@ public MergeOrchestratorConfigService(ProcessConfigRepository processConfigRepos
         this.networkStoreService = networkStoreService;
     }
 
-    List<ProcessConfig> getConfigs() {
-        return processConfigRepository.findAll().stream().map(this::toProcessConfig).collect(Collectors.toList());
+    public List<ProcessConfig> getConfigs() {
+        return self.doGetConfigs().stream().map(this::toProcessConfig).collect(Collectors.toList());
```

Comment:
we can avoid self invocation, by annotating getConfigs as transactional

Reply:
Done